src/main/java/seedu/address/model/person/Block.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's Block number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBlock(String)}
+ */
+public class Block {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Block should only contain alphabets";
+    public static final String VALIDATION_REGEX = "[a-zA-Z]";
```

Comment:
hold up, so block B and block b are 2 different blocks?

Reply:
Will add a toUpper() to enforce caps

Reply:
Thanks